src/routes/verify/[verifyToken]/+page.svelte

```diff
@@ -0,0 +1,91 @@
+<script>
+    // Framework imports
+    import { onMount } from "svelte";
+    import { urls, tryFetchingRestricted, toaster } from "$lib/util.js";
+    import { getToastStore, ProgressRadial } from "@skeletonlabs/skeleton";
+    import { goto } from "$app/navigation";
+
+    // Definitions
+    // Constants
+    const toastStore = getToastStore();
+
+    // Toast settings
+    const successToast = {
+        message: "Verifizierung erfolgreich",
+        bg: "primary",
+    };
+
+    const errorToast = {
+        message: "Verfizierung fehlgeschlagen",
+        bg: "error",
+    };
+
+    const serverErrorToast = {
+        message: "Fehler auf der Serverseite",
+        bg: "error",
+    };
+    
+    // Variables
+    export let data;
+
+    // Initialize verfied on default empty promise
+    let verified = new Promise(async (resolve, reject) => {});
+
+    // Functions
+    async function verifyTheToken() {
+        console.log(data.verifyToken);
+        verified = new Promise(async (resolve, reject) => {
+            try {
+                const response = await tryFetchingRestricted(
+                    urls.get.verifyUserToken + data.verifyToken,
+                );
+                if (response.ok) {
+                    resolve(await response.json());
+                    toastStore.trigger(toaster(successToast));
+                    goto("/login");
+                } else if (response.status == 400) {
```

Comment:
```suggestion
                } else if (response.status === 400) {
```
Sonst haben wir meist `===` benutzt, oder?

Reply:
Ehh bei den statuscode Überprüfungen bisher nicht 🤔